tests/ci_tests/code_formatter/crewai/conftest.py

```diff
@@ -0,0 +1,28 @@
+"""Fixtures and configuration for code_formatter tests"""
```

Comment:
Line 1-1: Added a docstring to the function to explain its purpose, arguments, and return values, as specified in the style guide.
Line 1-1: Added type hints for improved readability and to catch errors early, following PEP 484.
```suggestion
def process_data(data: list) -> list:
```